src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/XmlSerFormatAttrServices.cs

```diff
@@ -0,0 +1,349 @@
+            return i + j;
+        }
+
+        [OperationBehavior]
+        [XmlSerializerFormat(Style = OperationFormatStyle.Rpc, Use = OperationFormatUse.Literal)]
+        public int Sum(IntParams par)
+        {
+            return par.p1 + par.p2;
+        }
+
+        [OperationBehavior]
+        [XmlSerializerFormat(Style = OperationFormatStyle.Rpc, Use = OperationFormatUse.Literal)]
+        public float Divide(FloatParams par)
+        {
+            return (float)(par.p1 / par.p2);
+        }
+
+        [OperationBehavior]
+        [XmlSerializerFormat(Style = OperationFormatStyle.Rpc, Use = OperationFormatUse.Literal)]
+        public string Concatenate(IntParams par)
+        {
+            return string.Format("{0}{1}", par.p1, par.p2);
+        }
+
+        [OperationBehavior]
+        [XmlSerializerFormat(Style = OperationFormatStyle.Rpc, Use = OperationFormatUse.Literal)]
+        public void DoSomething(IntParams par)
+        {
+            Console.WriteLine("From RpcLitSingleNsService inside DoSomething method, params: {0} {1}", par.p1, par.p2);
+        }
+
+        [OperationBehavior]
+        [XmlSerializerFormat(Style = OperationFormatStyle.Rpc, Use = OperationFormatUse.Literal)]
+        public DateTime GetCurrentDateTime()
+        {
+            return DateTime.Now;
+        }
+
+        [OperationBehavior]
+        [XmlSerializerFormat(Style = OperationFormatStyle.Rpc, Use = OperationFormatUse.Literal)]
+        public byte[] CreateSet(ByteParams par)
+        {
+            return new byte[] { par.p1, par.p2 };
+        }
+    }
+
+    [ServiceBehavior(IncludeExceptionDetailInFaults = true)]
+    public class DocLitSingleNsService : ICalculator
+    {
+        [OperationBehavior]
+        [XmlSerializerFormat(Style = OperationFormatStyle.Document, Use = OperationFormatUse.Literal)]
+        public int Sum2(int i, int j)
+        {
+            return i + j;
+        }
+
+        [OperationBehavior]
+        [XmlSerializerFormat(Style = OperationFormatStyle.Document, Use = OperationFormatUse.Literal)]
+        public int Sum(IntParams par)
+        {
+            return par.p1 + par.p2;
+        }
+
+        [OperationBehavior]
+        [XmlSerializerFormat(Style = OperationFormatStyle.Document, Use = OperationFormatUse.Literal)]
+        public float Divide(FloatParams par)
+        {
+            return (float)(par.p1 / par.p2);
+        }
+
+        [OperationBehavior]
+        [XmlSerializerFormat(Style = OperationFormatStyle.Document, Use = OperationFormatUse.Literal)]
+        public string Concatenate(IntParams par)
+        {
+            return string.Format("{0}{1}", par.p1, par.p2);
+        }
+
+        [OperationBehavior]
+        [XmlSerializerFormat(Style = OperationFormatStyle.Document, Use = OperationFormatUse.Literal)]
+        public void DoSomething(IntParams par)
+        {
+            Console.WriteLine("From DocLitSingleNsService inside DoSomething method, params: {0} {1}", par.p1, par.p2);
```

Comment:
Console output on server side won't help much. I'd prefer to remove all `Console.WriteLine`.

Is this test testing return type being `void`? I think we can do something like this,
```c#
public void SetIntParamsProperty(IntParams par);
public IntParams GetIntParamsProperty();
```
We first call `SetIntParamsProperty` to set the value on server side and then call `GetIntParamsProperty` to retrieve the value and do verification.

Reply:
This applies to other test methods with `void` return type.

Reply:
Test updated.